src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackV2VolumePluginTest.java

```diff
@@ -84,53 +81,73 @@ public void setUp() throws Exception {
         this.tokenDefault = new Token(FAKE_ACCESS_ID, tokenUser, new Date(), attributes);
     }
```

Comment:
@llvieira I'm biased agains spies. Why do we need them in our test cases (line 68)?